libs/oef-base/include/oef-base/threading/Task.hpp

```diff
@@ -87,6 +87,11 @@ class Task : public std::enable_shared_from_this<Task>
     return std::dynamic_pointer_cast<Derived>(shared_from_this());
   }
 
+  std::shared_ptr<Taskpool> GetTaskPool()
```

Comment:
Is this used anywhere?

Reply:
Removed it (pushed a bunch of stuff to wrong branch earlier!!)